src/components/Family/First.vue

```diff
@@ -87,6 +86,9 @@ export default {
   max-width: 30%;
   padding: 0 4px;
 }
+img{
+  height:230px;
+}
```

Comment:
+1
Other than this, I do not think the changes done in the other files to be needed.